docs/useful-tips-on-carbondata.md

```diff
@@ -125,6 +125,10 @@
     TBLPROPERTIES ('SORT_COLUMNS'='Dime_1, HOST, MSISDN')
   ```
 
+  **NOTE:**
+  + BloomFilter can be created to enhance performance for queries with precise equal/in conditions. You can find more information about it in BloomFilter datamap document.
+
```

Comment:
please add one link to bloomfilter datamap document.

Reply:
OK